katacomb/katacomb/conf/uvblavg_MKAT.yaml

```diff
@@ -1,6 +1,4 @@
 # Default options for UVBlAvg
 FOV: 1.0         # Field of view (deg)
-maxInt: 1.0      # Maximum integration (min)
+maxInt: 0.5      # Maximum integration (min)
```

Comment:
If this is linked to some other parameter in self-calibration it might be a good idea to add a comment documenting this so in future people know not to alter one without altering the other.

Reply:
Good point!